app/packages/embeddings/src/utils.ts

```diff
@@ -6,7 +6,7 @@ export type Filters = {
   [key: string]: {};
 } | null;
 
-const PATCHES_VIEW_STAGE = "fiftyone.core.stages.PatchesView";
+const PATCHES_VIEW_STAGE = "fiftyone.core.stages.ToPatches";
```

Comment:
This fixes a slightly different issue, where the wrong view stage was used to determine whether samples view was using `ToPatches`.